Update the E2E test with scalardl-java-client-sdk

scalar-boney opened the pull request:
Use [scalardl-java-client-sdk-****.zip](https://github.com/scalar-labs/scalardl-java-client-sdk/releases) file instead of creating a client from the Scalar repo.

ypeckstadt reviewed (approved):
Code wise it looks good to me. Just have to make sure that it is allowed to use the Scalar DL client source code like this.

supl reviewed (approved):
It looks good to me and I think we can merge it.

But, somehow I am thinking maybe we should find a more convenient way to use these Scalar DL client-side tools.

feeblefakie reviewed (commented):
Why not download the zip instead of copying the files?
Managing duplicate files is not very maintainable.
https://github.com/scalar-labs/scalardl-java-client-sdk/releases/tag/v3.6.0

scalar-boney commented:
> Why not download the zip instead of copying the files? Managing duplicate files is not very maintainable. https://github.com/scalar-labs/scalardl-java-client-sdk/releases/tag/v3.6.0

Sorry, I forgot the `Scalar DL Java Client SDK` availability in the `scalar-labs/scalardl-java-client-sdk` repo.

I have removed the new duplicated tool and updated the e2e test.

Thank you for your suggestion.

@feeblefakie @ypeckstadt @supl san, 
Could you please take a look when you get a chance?

feeblefakie commented on .github/workflows/e2e.yml:
```diff
@@ -51,10 +43,11 @@ jobs:
           ./gradlew build
 
-      - name: Build Scalar DL Client
+      - name: Set up Scalar DL Java Client SDK
```
```suggestion
      - name: Download Scalar DL Java Client SDK
```

feeblefakie reviewed (approved):
LGTM! Thank you!

supl reviewed (approved):
LGTM.
Thank you.

ypeckstadt reviewed (approved):
LGTM. Thank you.
